build-tools/debian-metadata/xamarin.android.install

```diff
@@ -0,0 +1,2 @@
+bin/Debug/bin/xabuild		/usr/bin
```

Comment:
OK, this is now built using $(CONFIGURATION). which works in local testing, but i expect was the wrong choice for a real build.